rust/perspective-python/src/client/client_sync.rs

```diff
@@ -40,7 +40,7 @@ pub struct PySyncClient(PyClient);
 #[pymethods]
 impl PySyncClient {
     #[new]
-    pub fn new(callback: Py<PyFunction>) -> PyResult<Self> {
+    pub fn new(callback: Py<PyAny>) -> PyResult<Self> {
```

Comment:
TODO check for call

Reply:
The previous version of Perspective did not do this, and Python has adequate error handling for this already.

Reply:
Its quite bit uglier now though:
```raw
perspective.PerspectivePyError: External error: PyErr { type: <class 'TypeError'>, value: TypeError("'<Type>' object is not callable"), traceback: Some(<traceback object at 0x15c1e5780>) }
```

Reply:
I've fixed this error message on #2659